torchtext/models/multiheadattention.py

```diff
@@ -0,0 +1,236 @@
+import torch
+from torch._jit_internal import Tuple, Optional
```

Comment:
now that python2 is not supported anymore, we can use `from typing import Tuple, Optional` instead, as `typing ` is available in Python3 by default